hack/genpkgs/main.go

```diff
@@ -4,26 +4,23 @@ import (
 	"encoding/csv"
 	"fmt"
 	"log"
-	"math/rand"
 	"os"
 	"strconv"
-	"time"
 
 	"github.com/google/uuid"
 )
 
 const (
 	// How many records we want to generate.
-	datasetSize = 2500000
+	datasetSize = 55
 
 	// Size of each batch that we write to the CSV.
 	batchSize = 100
 )
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
-
 	w := csv.NewWriter(os.Stdout)
+	defer w.Flush()
```

Comment:
Nice catch! But, instead of using defer, I'd do it right after the for loop, before calling `w.Error()`:

> func (w *csv.Writer) Error() error
>
> Error reports any error that has occurred during a previous [Writer.Write] or [Writer.Flush].